sklearn/svm/tests/test_sparse.py

```diff
@@ -39,12 +39,21 @@
 
 
 def check_svm_model_equal(dense_svm, sparse_svm, X_train, y_train, X_test):
-    dense_svm.fit(X_train.toarray(), y_train)
+    if isinstance(dense_svm, svm.OneClassSVM):
```

Comment:
I don't think these cases are necessary. `OneClassSVM` should accept `y=None` quietly.